homeassistant/components/thethingsnetwork/__init__.py

```diff
@@ -33,10 +37,57 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Initialize of The Things Network component."""
-    conf = config[DOMAIN]
-    app_id = conf.get(CONF_APP_ID)
-    access_key = conf.get(CONF_ACCESS_KEY)
 
-    hass.data[DATA_TTN] = {TTN_ACCESS_KEY: access_key, TTN_APP_ID: app_id}
+    if DOMAIN in config:
+        ir.async_create_issue(
+            hass,
+            DOMAIN,
+            "manual_migration",
+            breaks_in_ha_version="2024.11.0",
```

Comment:
```suggestion
            breaks_in_ha_version="2024.12.0",
```
I do have some spare time today in the bus, i think we can make the .6

Reply:
Great - moving to 2024.6.0